### PR TITLE
Upgrade cmake to 3.13.1 in prereqs and Dockerfile

### PR DESCRIPTION
**Pull request by dragoshsch**
- Added install steps to hardcode CMake 3.13.1 in install-prereqs
- Added install steps to upgrade CMake to 3.13.1 in Dockerfile

### CODE REVIEW

**Comment by dragoshsch** [comment]
bors try

**Comment by dragoshsch** [comment]
bors try

File: scripts/install-prereqs [hunk 1]
```diff
@@ -77,3 +77,11 @@ EOF
 
     apt-get -y install clang-7
 fi
+
+# Install cmake 3.13.1
```
Comment (andschwa) [inline scripts/install-prereqs]: Could we change this just slightly to do the following:

if (cmake is not present at all) then (install via download)
if (cmake is installed but not <= 3.13.1) then (exit with an error message)

That way we only install if the user hadn't already installed it somehow, but we still check that it installed correctly (regardless of how it was installed).

Thank you!

**Comment by dragoshsch** [comment]
bors try

File: scripts/install-prereqs (hunk 1 above)
Comment (dragoshsch) [reply scripts/install-prereqs]: Changes added!

**Comment by dragoshsch** [comment]
@johnkord added! also fixed my quote typo :)

**Comment by dragoshsch** [comment]
bors try

**Pull request merged**